### PR TITLE
objcopy: add page 

### PR DESCRIPTION
**Pull request by pranav10780**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- GNU objcopy (GNU Binutils for Ubuntu) 2.38

### CODE REVIEW

File: pages/linux/objcopy.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# objcopy
+
+> Copy the contents of an object file to another.
```
Comment (kbdharun) [inline pages/linux/objcopy.md:3]:
```suggestion
> Copy the contents of an object file to another file.
```

File: pages/linux/objcopy.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://manned.org/objcopy>.
+
+- Copy data to another file:
+
+`objcopy {{path/to/source_file}} {{path/to/target_file}}`
+
+- Translate object files from one format to another:
+
+`objcopy --input-target={{inputformat}} --output-target={{output_format}} {{path/to/source_file}} {{path/to/target_file}}`
```
Comment (kbdharun) [inline pages/linux/objcopy.md:12]:
```suggestion
`objcopy --input-target={{input_format}} --output-target={{output_format}} {{path/to/source_file}} {{path/to/target_file}}`
```

File: pages/linux/objcopy.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Strip all symbol information from file:
+
+`objcopy --strip-all {{path/to/source_file}} {{path/to/target_file}}`
+
+- Strip debugging information from file:
```
Comment (kbdharun) [inline pages/linux/objcopy.md:18]:
```suggestion
- Strip all symbol information from the file:

`objcopy --strip-all {{path/to/source_file}} {{path/to/target_file}}`

- Strip debugging information from the file:
```

File: pages/linux/objcopy.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`objcopy --strip-debug {{path/to/source_file}} {{path/to/target_file}}`
+
+- Copy a specific section from the source to destination:
```
Comment (kbdharun) [inline pages/linux/objcopy.md:22]:
```suggestion
- Copy a specific section from the source file to the destination file:
```

File: pages/linux/objcopy.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`objcopy --only-section={{section}} {{path/to/source_file}} {{path/to/target_file}}`
```
Comment (kbdharun) [inline pages/linux/objcopy.md:24]:
```suggestion
`objcopy --only-section {{section}} {{path/to/source_file}} {{path/to/target_file}}`
```
Checked the docs locally and `=` here isn't required for the input to work.

Ref.

```bash
kbdharun@server:~# objcopy -h
Usage: objcopy [option(s)] in-file [out-file]
 Copies a binary file, possibly transforming it in the process
 The options are:
  -I --input-target <bfdname>      Assume input file is in format <bfdname>
  -O --output-target <bfdname>     Create an output file in format <bfdname>
  -B --binary-architecture <arch>  Set output arch, when input is arch-less
  -F --target <bfdname>            Set both input and output format to <bfdname>
     --debugging                   Convert debugging information, if possible
  -p --preserve-dates              Copy modified/access timestamps to the output
  -D --enable-deterministic-archives
                                   Produce deterministic output when stripping archives (default)
  -U --disable-deterministic-archives
                                   Disable -D behavior
  -j --only-section <name>         Only copy section <name> into the output
     --add-gnu-debuglink=<file>    Add section .gnu_debuglink linking to <file>
... (The below options aren't relevant for this review comment)
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by pranav10780** [comment]
Should I make a new pr is there a better way of doing things because I see it getting out of hand fast

**Comment by kbdharun** [comment]
> Should I make a new pr is there a better way of doing things because I see it getting out of hand fast

No, you can apply the suggestions directly here, feel free to check this out https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

**Review by gutjuri — APPROVED** [review]
LGTM after the above suggestions! Thanks for your contribution!

**Comment by pranav10780** [comment]
all suggestions have been added

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**